src/actions/actions_custom.py

```diff
@@ -0,0 +1,30 @@
+from talon import Module, app, actions
+from ..csv_overrides import init_csv_and_watch_changes
+
+custom_action_defaults = {}
+
+
+mod = Module()
+mod.list(
+    "cursorless_custom_action",
+    desc="Supported custom actions for cursorless navigation",
+)
+
+
+@mod.capture(rule="{user.cursorless_custom_action}")
+def cursorless_custom_action(m) -> callable:
+    return lambda targets: actions.user.cursorless_single_target_command(
```

Comment:
Using a `lambda` doesn't feel very Pythonic (feels more like typescript 😊).  But it is clean and compact so let's stick with it for now

Reply:
I'm always aiming for code clarity. There is a lambda in python and by the universe I will use it! :D